onnx/checker.cc

```diff
@@ -328,14 +366,22 @@ void check_node(
   }
 
   // Put the removed experimental ops here
-  static std::set<std::string> experimental_ops = {"ATen", "Affine",
-    "ConstantFill", "Crop", "DynamicSlice", "GRUUnit", "GivenTensorFill",
-    "ImageScaler", "ParametricSoftplus", "Scale", "ScaledTanh"};
+  static std::set<std::string> experimental_ops = {"ATen",
```

Comment:
Do we still have experimental ops?

Reply:
I didn't touch this. It was reformatted automatically by VisualStudio using the clang format. (I believe this might be used for backward-compatibility … Ke would know.)

Reply:
Ah.. For BC, yes, we need to know those operator names.